src/HashBus.Viewer.MostMentioned/LeaderboardView.cs

```diff
@@ -180,9 +182,9 @@ public Task RunAsync()
 
                     var tokens = new[]
                     {
-                        $"\r{padding}Total {itemsName}: ".DarkGray(),
-                        $"{currentLeaderboard?.Count ?? 0:N0}".Color(totalColor),
-                        $" · Refreshing in {timeLeft.Humanize()}...".DarkGray()
+                        $"\r{padding}Fork me at ".DarkGray(),
+                        "github.com/Particular/HashBus".Blue(),
+                        $" · {timeLeft.TotalSeconds}".DarkGray(),

```

Comment:
If you dropped, fork me at does "Refresh" fit? Alternatively a symbol like ↻ might indicate the intent better.